card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
I'm not sure you quite understand that there really isn't much to decko-rails.  Or rather, there isn't much in wagn -> decko, so I'm not sure I understand what you want here.

I don't see the problem with implementing the 2nd case as an extension that you don't really see unless you use it a certain way.  Both cases are a rails app, but in one you don't open up the app at all.  Extensions are modules within the app.

I think I'm finally seeing what you mean about just having our migrations.  The generated migration doesn't have to change, or rather, it only changes when the schema changes.  In fact, if we just load the schema by running the *initial* migration into a database with no tables instead of the traditional rails schema load, that solves the DRY issue with the base schema.  It's just a migration that creates an empty deck.

I don't think we need a decko-rails command.  That still should be a generator (I guess you want it to be called wiki (rails g wiki [card])), but it doesn't have much to do unless you make it actually edit (and backup) the files it needs to config.  I'm trying to reduce the number of files to a minimum.  I will also try to get it to attach our migrations to the base tasks (db:migrate).

I never wanted to duplicate things, but I also don't really want to reference wagn tasks from decko-rails.  Can we move them to the decko now? (or will it be deck for rake tasks?)  So decko:migrate and sub-tasks should be the same for both cases (aka wagn:migrate).